databaseFunctions.js

```diff
@@ -0,0 +1,72 @@
+const { db, end } = require("./database.js");
+
+// Takes in task name for creation and outputs the id of the created task
```

Comment:
Can we convert this comment to a properly-formatted [JSDoc](https://jsdoc.app/about-getting-started) comment?